src/main/java/uk/gov/companieshouse/overseasentitiesapi/converter/TransformerFactory.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.overseasentitiesapi.converter;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.overseasentitiesapi.model.SchemaVersion;
+
+import javax.annotation.PostConstruct;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+@Service
+public class TransformerFactory {
```

Comment:
i'd rename the factory now to DocumentTransformerFactory as it 'produces' DocumentTransformers

Reply:
Updated the name for the class and the test class